modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheMapEntry.java

```diff
@@ -6438,7 +6438,14 @@ else if (newSysTtl == CU.TTL_ZERO) {
                 CacheLazyEntry<Object, Object> interceptEntry =
                     new CacheLazyEntry<>(cctx, entry.key, null, oldVal, null, keepBinary);
 
-                Object interceptorVal = cctx.config().getInterceptor().onBeforePut(interceptEntry, updated0);
+                Object interceptorVal = null;
+
+                try {
```

Comment:
Same question as in #8195 - do we really need this change here? If yes, can we add a corresponding test?

Reply:
I think yes, without this change the thin client can make a failed server node.
IgniteBinaryTest#testBinaryWithNotGenericInterceptor checked it.